client-proxy/database/database.go

```diff
@@ -0,0 +1,45 @@
+package database
+
+import (
+	"database/sql"
+	"github.com/OpenLineage/OpenLineage/client-proxy/config"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+)
+
+type Database struct {
+	db *sql.DB
+}
+
+type IDatabase interface {
+}
+
+func New(conf config.Config) *Database {
+	db, err := sql.Open("sqlite3", conf.SqliteDatabasePath)
+	if err != nil {
+		log.Fatalln("Cannot open database:", err)
+	}
+
+	log.Println("Initializing DB")
+	InitDB(db)
+	return &Database{db: db}
+}
+
+func InitDB(db *sql.DB) {
```

Comment:
Dows `InitDB()` need to be a public function? Otherwise, I would make it private `initDB()`

Reply:
Ah, is that a go convention, starting with lower case for private? Good to know. Thank you.

Reply:
Sure is ;)